tools/integration_tests/run_e2e_tests.sh

```diff
@@ -239,6 +239,25 @@ function run_e2e_tests_for_hns_bucket(){
    return 0
 }
 
+function run_e2e_tests_for_anywhere_cache(){
+   local test_bucket_name="golang-grpc-test-lankita-anywhere-cache-test" ## TODO: adding temporary bucket name for testing purpose
```

Comment:
Can we set test bucket name with `golang-grpc-test-gcsfuse-anywhere-cache-e2e-tests` ?

Reply:
yes, in this PR just trying to run the test and see if cache is being used or not, will update final bucket name once cache usage is confirmed.